README.md

```diff
@@ -238,4 +238,47 @@ How to check status and read connection metrics:
     curl --cacert test-keys/root.crt https://localhost:6060/_metrics
```

Comment:
nit: is it possible to escape the underscores in the URLs so that they don't italicize the rest of the README?